src/lib.rs

```diff
@@ -256,6 +256,21 @@ impl<T> Event<T> {
     ///
     /// In certain cases, this function emits a `SeqCst` fence before notifying listeners.
     ///
+    /// ## Return Value
+    ///
+    /// This function returns the number of [`EventListener`]s that were notified by this
+    /// call. However, in some cases, the notification will be delayed and this function will
+    /// return `0` instead of the number of listeners actually notified. Therefore, the return
+    /// value of this function should not be relied upon for soundness and should be used only
+    /// as a hint.
+    ///
+    /// The cases where `0` will be spuriously returned include:
+    ///
+    /// - If the `std` feature is enabled, no spurious returns are possible, since the `std`
```

Comment:
sorry to nitpick but that's a strange way of putting it. You say you're going to list cases where `0` will be returned and then the first item is a case where `0` will not be returned.

Reply:
LGTM otherwise. :+1: